smallest_multiple.js

```diff
@@ -6,9 +7,18 @@
  * @return { Number }         Lowest Positive Number that is evenly divisible by all numbers
  *                            between 1 and `ceiling`
  */
-module.exports = function( ceiling ) {
-  // do work here
-
-
-  return 0;
-};
\ No newline at end of file
+ module.exports = function( ceiling ) {
+    for(var i = 1; i <= 10000000000000; i++){
```

Comment:
Could use Infinity instead or a boolean flag to signal when to end the outer for loop.